crates/oxc_linter/examples/queries/next_no-unwanted-polyfillio/the_rule.notron

```diff
@@ -0,0 +1,116 @@
+InputQuery (
```

Comment:
This query was never finished, so it's still in the old format, should we just scratch it for now and remake it when it's ready?